src/main/kotlin/no/nav/familie/tilbake/behandlingskontroll/BehandlingskontrollService.kt

```diff
@@ -356,7 +356,7 @@ class BehandlingskontrollService(
 
         val sisteUtførteSteg =
             stegstilstand.filter { Behandlingsstegstatus.erStegUtført(it.behandlingsstegsstatus) }
-                .maxByOrNull { it.sporbar.endret.endretTid }!!.behandlingssteg
+                .maxByOrNull { it.behandlingssteg.sekvens }!!.behandlingssteg
```

Comment:
Hvordan blir dette da når man hoppe frem og tilbake på steg?

Reply:
Kanskje litt overengineering, men kunne vi sjekket tid og evt sekvens?;

```
maxWithOrNull(compareBy(
                { it.sporbar.endret.endretTid },
                { it.behandlingssteg.sekvens }
            ))
```

Reply:
Har nå testet i preprod og resultatet av kodeendringen ble at hvis man hopper tilbake på steg, så blir stegene etter grået ut, selv om infoen fortsatt finnes.
<img width="1714" alt="Screenshot 2024-05-13 at 12 58 36" src="https://github.com/navikt/familie-tilbake/assets/42737566/03ec4c31-3ab9-488b-b076-08f6ea376a30">

Dersom man fyller ut alt til og med vedtak, og deretter går tilbake til steg 1, så må man bekrefte tidligere lagt inn info på steg 2 og 3 for å komme tilbake til 4. Velger derfor å gå for Charlies løsning.